examples/creatives/nonlinear_overlay.html

```diff
@@ -5,9 +5,41 @@
       <script src="nonlinear_overlay.js"> </script>
     </head>
     <style>
+      button {
```

Comment:
Did the other example ads do css this way or did they use an external style sheet?  If they did it this way then its fine to maintain this,  otherwise id say to use an external sheet.

Reply:
the other ads do it this way- just tried to stay consistent